arch/xtensa/core/xtensa-asm2-util.S

```diff
@@ -298,27 +298,40 @@ _handle_excint:
 
 DEF_EXCINT 1, _handle_excint, xtensa_excint1_c
 
-#if XCHAL_NMILEVEL >= 2
+/* In code below we are using XCHAL_NMILEVEL and XCHAL_NUM_INTLEVELS
+ * (whichever is higher), since not all Xtensa configurations support
+ * NMI. In such case we will use XCHAL_NUM_INTLEVELS.
+ */
+#if XCHAL_HAVE_NMI
+#define _MAX_INTR_LEVEL XCHAL_NMILEVEL
```

Comment:
Could you remove the underscore prefix? We are trying not to prefix any symbols or identifiers with underscore.

Reply:
Sure thing!